accloudtant/aws/prices.py

```diff
@@ -80,7 +86,8 @@ def print_prices(instances=None, region='us-east-1'):
             },
         },
     }
-    headers = ['Type',
+    headers = ['Region',
```

Comment:
The CLI is requiring to specify the refion, so I don't see the need to show this.